<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWnd</class>
 <widget class="QMainWindow" name="MainWnd">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>799</width>
    <height>657</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Remote control</string>
  </property>
  <widget class="QWidget" name="cw">
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="spacing">
     <number>1</number>
    </property>
    <property name="margin">
     <number>1</number>
    </property>
    <item>
     <widget class="QGraphicsView" name="view">
      <property name="sizePolicy">
       <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>512</width>
        <height>256</height>
       </size>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>799</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QDockWidget" name="cmdDw">
   <property name="sizePolicy">
    <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="features">
    <set>QDockWidget::DockWidgetFloatable|QDockWidget::DockWidgetMovable</set>
   </property>
   <property name="windowTitle">
    <string>Command</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>8</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents_2">
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <property name="spacing">
      <number>0</number>
     </property>
     <property name="margin">
      <number>0</number>
     </property>
     <item>
      <widget class="QtLua::Console" name="console" native="true">
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <property name="spacing">
         <number>0</number>
        </property>
        <property name="margin">
         <number>0</number>
        </property>
       </layout>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QDockWidget" name="platformDw">
   <property name="features">
    <set>QDockWidget::DockWidgetFloatable|QDockWidget::DockWidgetMovable</set>
   </property>
   <property name="windowTitle">
    <string>Platform</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>8</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents_3">
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <widget class="AnalogPad" name="joy1" native="true">
       <property name="minimumSize">
        <size>
         <width>128</width>
         <height>128</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="AnalogPad" name="joy2" native="true">
       <property name="minimumSize">
        <size>
         <width>128</width>
         <height>128</height>
        </size>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="iconSize">
    <size>
     <width>64</width>
     <height>64</height>
    </size>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="image"/>
   <addaction name="connect"/>
   <addaction name="exec"/>
   <addaction name="send"/>
   <addaction name="valve"/>
   <addaction name="separator"/>
   <addaction name="help"/>
  </widget>
  <widget class="QDockWidget" name="manipulatorDw">
   <property name="features">
    <set>QDockWidget::DockWidgetFloatable|QDockWidget::DockWidgetMovable</set>
   </property>
   <property name="windowTitle">
    <string>Manipulator</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>8</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents">
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="AnalogPad" name="joy3" native="true">
       <property name="minimumSize">
        <size>
         <width>128</width>
         <height>128</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="AnalogPad" name="joy4" native="true">
       <property name="minimumSize">
        <size>
         <width>128</width>
         <height>128</height>
        </size>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <action name="image">
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/images/camera.png</normaloff>:/images/camera.png</iconset>
   </property>
   <property name="text">
    <string>Get image</string>
   </property>
   <property name="toolTip">
    <string>Get image from attached web camera</string>
   </property>
  </action>
  <action name="connect">
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/images/network.png</normaloff>:/images/network.png</iconset>
   </property>
   <property name="text">
    <string>Connect/reconnect</string>
   </property>
   <property name="toolTip">
    <string>Connect/reconnect to remote host application</string>
   </property>
  </action>
  <action name="exec">
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/images/ctrl.png</normaloff>:/images/ctrl.png</iconset>
   </property>
   <property name="text">
    <string>Exec</string>
   </property>
   <property name="toolTip">
    <string>Invoke local script file</string>
   </property>
  </action>
  <action name="send">
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/images/send.png</normaloff>:/images/send.png</iconset>
   </property>
   <property name="text">
    <string>Send</string>
   </property>
   <property name="toolTip">
    <string>Send script file</string>
   </property>
  </action>
  <action name="help">
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/images/cap.png</normaloff>:/images/cap.png</iconset>
   </property>
   <property name="text">
    <string>Help</string>
   </property>
   <property name="toolTip">
    <string>Display help window!</string>
   </property>
   <property name="shortcut">
    <string>F1</string>
   </property>
  </action>
  <action name="valve">
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/images/valve.png</normaloff>:/images/valve.png</iconset>
   </property>
   <property name="text">
    <string>Valve</string>
   </property>
   <property name="toolTip">
    <string>Valve test window</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QtLua::Console</class>
   <extends>QWidget</extends>
   <header>QtLua/Console</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>AnalogPad</class>
   <extends>QWidget</extends>
   <header>analogpad.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="images.qrc"/>
 </resources>
 <connections/>
</ui>
